<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PNGWindow</class>
 <widget class="QMainWindow" name="PNGWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select .png file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOriginalImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDitherImage">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Image after Ordered Dithering</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBayer">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8 x 8 Bayer</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonFloydSteinberg">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Floyd-Steinberg</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
